src/__tests__/demo.js

```diff
@@ -37,13 +37,13 @@ window.adobeDataLayer.push({
   }
 });
 
-window.adobeDataLayer.getState().component.carousel;
+// window.adobeDataLayer.getState().component.carousel;
```

Comment:
Why are these commented out?

Reply:
Due to js lint going through this file we should not have `unused variables`. It is just example of what you can do with DL and is not required by any other piece of the code.

Reply:
Instead of commenting out I would adapt the code to comply with js lint.
And also I would write it with a path argument now that we support it:
window.adobeDataLayer.getState('component.carousel');